src/site/stages/build/plugins/rewrite-drupal-pages.js

```diff
@@ -9,10 +9,15 @@ function rewriteDrupalPages(options) {
       const replacements = Object.keys(files)
         .filter(fileName => files[`drupal/${fileName}`])
         .map(fileName => ({
-          from: `"/${fileName.replace('index.html', '')}"`,
+          from: `"/${fileName.replace('/index.html', '')}"`,
```

Comment:
This is just something I found - the sidebar links weren't updating with this change. Might have to mess with this to be sure this is the right thing.